# FSM_state

uint8 mode_iiwa7
uint8 mode_iiwa14
bool isHit
time hit_time